documentation/Tutorials.md

```diff
@@ -0,0 +1,192 @@
+# Tutorials
```

Comment:
Maybe change the documentation of the functions, which means to add the 'link' feature as used in the tutorials wiki

Reply:
what do you mean?